angular-ui/src/app/footer/footer.component.html

```diff
@@ -1 +1,5 @@
-<p>footer works!</p>
+<p>Thanks for visiting our site. This site was created by the Angular in the Outfield group of the 2023 LaunchCode Liftoff Project</p>
```

Comment:
You might wrap these p elements with a footer element. This might make styling the footer easier later. This is an awesome base for us to start with!